packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/PickButton/PickButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { useSignals, useStore } from 'app/store';
+import { Button } from './elements';
+
+export const PickButton = observer(() => {
+  const {
+    explore: { pickSandboxModal },
+  } = useSignals();
+  const {
+    editor: {
+      currentSandbox: { id, title, description, owned },
+    },
+  } = useStore();
+
+  return (
+    <Button
+      onClick={() => {
+        pickSandboxModal({
```

Comment:
@SaraVieira I think you should extract this into a separate variable to make the return a bit cleaner.

Reply:
Good point! Moved :)